app/controllers/merchant_dashboard_controller.rb

```diff
@@ -0,0 +1,6 @@
+class MerchantDashboardController < ApplicationController
+    def show 
+        binding.pry
+    end 
```

Comment:
I think we can delete this file.

Reply:
You might be right, Ill take a look as we do the next few user stories